app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def not_found
```

Comment:
ApplicationHelper zawierał metody przeznaczone dla controllerów i był dołączany do controllerów: http://en.wikibooks.org/wiki/Ruby_on_Rails/ActionView/Custom_Helpers#Application_Helper

Jeśli ta metoda jest przeznaczona do działania w controllerach to należy ją przenieść po prostu do ApplicationController - będzie dostępna we wszystkich controllerach. Dla jednej metody nie ma co robić całego modułu - chyba, że potrafimy wyznaczyć odpowiedzialność takiego modułu.